### DISCUSSION TITLE
why not ship ML libraries with ImageTooBox FOSS?

### DISCUSSION THREAD

**Discussion by devSJR** [T8RIN/ImageToolbox#802] [General]
I have to admit that I only briefly covered the topic, but I wonder why it is not possible to provide the Machine Learning Tools from Google with the FOSS version of ImageToolBox. 
https://developers.google.com/ml-kit
If I see this correctly, these are distributed under the Apache license. Should this not be compatible with ImageToolbox? I suspect that I am overlooking something here.

**Comment by devSJR** [comment]
Wait, just found that https://github.com/T8RIN/ImageToolbox/discussions/599. Seems to answer the question.

**Comment by T8RIN** [comment]
This question is better to ask from @izzysoft

**Comment by licaon-kter** [comment]
MLKit is not FOSS: https://mvnrepository.com/artifact/com.google.mlkit

/close this
